packages/angular_devkit/build_optimizer/src/purify/purify_spec.ts

```diff
@@ -27,17 +27,36 @@ describe('purify', () => {
 
   it('prefixes safe default imports with /*@__PURE__*/', () => {
     const input = tags.stripIndent`
-      /* harmony import */ var __WEBPACK_IMPORTED_MODULE_4_rxjs_Subject__ = __webpack_require__("rlar");
-      /* harmony import */ var __WEBPACK_IMPORTED_MODULE_4_rxjs_Subject___default = __webpack_require__.n(__WEBPACK_IMPORTED_MODULE_4_rxjs_Subject__);
+    /* harmony import */ var __WEBPACK_IMPORTED_MODULE_4_rxjs_Subject__ = __webpack_require__("rlar");
```

Comment:
this seems like an incorrect indentation change

Reply:
Fixed